Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
 # create a work dir and copy local files in
 WORKDIR /go/src/github.com/openshift/console-operator
 COPY . .
-RUN make build
+
+RUN make build WHAT="cmd/console" GOFLAGS="-brandokd"
```

Comment:
I would just have one `-ocp` flag, then fallback to OKD logo when that flag is not present. I don't think we need tags for every brand value.

Reply:
Ok, so when it comes to building the operator, we only want okd or ocp.  The other options will never be available through the build?

Reply:
Right, and I probably would only have one tag, `ocp`. Then we use OKD branding when `ocp` is not set.